.gitignore

```diff
@@ -34,6 +34,8 @@ coverage/
 testResults/junit.xml
 
 *-css.ts
+*.tsbuildinfo
+package-lock.json
```

Comment:
Curious why the lock files are now ignored?

Reply:
package-lock is an npm mechanism. Now that we use yarn, we should not use it. Yarn will even throw a warning.